app/lib/server/functions/validateName.js

```diff
@@ -0,0 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+
+import { settings } from '../../../settings';
+
+export const validateName = function(name) {
+	if (settings.get('Accounts_SystemBlockedUsernameList').includes(name.toLowerCase())) {
+		throw new Meteor.Error('invalid-name', `${ name } is a reserved name.`);
```

Comment:
in this case, since this function is almost "pure", it is better to use regular `Error` instead of `Meteor.Error`.. we should use `Meteor.Error` only on Meteor environments (like Meteor methods), otherwise always use regular Node exceptions